tools/ou_rs_amplitude_retune_compare.py

```
@@ -0,0 +1,188 @@
+
+
+def load(raw: Path) -> dict[tuple[str, int], dict[str, str]]:
+    with raw.open() as fh:
+        return {(r["scenario"], int(r["repetition"])): r for r in csv.DictReader(fh)}
+
+
+def paired(base: dict, rev: dict, keys: list, metric: str) -> tuple[float, float, float, float]:
+    """(baseline mean, revised mean, mean paired delta, 95% half-width)."""
+    b = [float(base[k][metric]) for k in keys]
+    r = [float(rev[k][metric]) for k in keys]
+    d = [x - y for x, y in zip(r, b)]
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)